### PR TITLE
Added information box when hovering over pokemon

### LINKED ISSUE #28: Display pokemon statistics retrieved from the json
Once the result is displayed, we can display stats of each pokemon on hovering over it's sprite.

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [ ] Assign this issue to me (I want to work on this).
- [ ] I will send my PR only after I'm assigned to this issue by any maintainer.
- [ ] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

### PR DESCRIPTION
**Pull request by jmeline**
Closes #28

#### By submitting this pull request I confirm I've read and complied with the below declarations.

- [x] I have tested the changes locally and they are functional.
- [x] I ensure that my commit message is in imperative manner.
- [x] I ensure that I have properly referenced the related issue.
- [x] I ensure that I sent this pull request after being assigned to the corresponding issue.
- [x] This pull request has a descriptive title. For example, `Add/Fix {Feature/Bug}`, not `Update README.md` or `Add new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.

2018-10-02-225505_3520x1080_scrot

### CODE REVIEW

**Comment by jmeline** [comment]
Tests will be added probably after work later tonight.

File: src/components/PokemonSprite.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PokeSprite from 'react-poke-sprites';
+import 'react-tippy/dist/tippy.css';
+import { Tooltip } from 'react-tippy';
+
+const Info = ({ info }) => {
+  return (
+    <React.Fragment>
+      <div>name: {info.species} </div>
```
Comment (jmeline) [inline src/components/PokemonSprite.js]: What other information would be interesting to look at? I will reformat the base stats display as it is just ugly json at the moment.

File: src/components/UserInput.js [hunk 2]
```diff
@@ -3,7 +3,8 @@ import { getPokemonInheritance } from '../helpers';
 import { BattlePokedex } from '../pokedex';
 import SkyLight from 'react-skylight';
 import FeelingHype from './FeelingHype';
-import PokeSprite from 'react-poke-sprites';
+// import PokeSprite from 'react-poke-sprites';
```
Comment (jmeline) [inline src/components/UserInput.js]: Comment will be removed

File: src/components/PokemonSprite.js (hunk 1 above)
Comment (anshumanv) [reply src/components/PokemonSprite.js]: What I was thinking of is 
`name, height, weight, eveything in base stats, and abilities`

Would that be too long? What do you think?

**Comment by anshumanv** [comment]
Hey @jmeline thanks for looking this up, replied to your review. :+1: :smile:

File: src/components/PokemonSprite.js (hunk 1 above)
Comment (jmeline) [reply src/components/PokemonSprite.js]: I tried a few different iterations and found a layout that I think works. If you have any feedback, I am happy to switch some things around.

File: src/components/PokemonSprite.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+const rowStyle = {
+  overflow: 'auto',
+  height: '100%',
+};
+
+const Stats = ({ baseStats }) => (
+  <React.Fragment>
+    <div> Stats: </div>
+    <div style={rowStyle}>
+      <div style={colStyle}>Hp: {baseStats.hp} </div>
```
Comment (jmeline) [inline src/components/PokemonSprite.js]: For some reason I couldn't seem to get regular className css imports to be applied to the tooltip. I ended up going with this route until I could figure out why. 

I imported the css above.
```import ../css/PokemonSprite.css```

Then I used it in these div tags but none of the styles were coming through. Upon inspecting the developer tools, under the styling tab in firefox, I verified that I could view my css. I am sure I am missing something extremely silly. If anyone can spot what I did wrong, please let me know.

Thanks

Comment (jmeline) [reply src/components/PokemonSprite.js]: nvm, I think it was caching related. Got it working.

**Comment by anshumanv** [comment]
This is amazing @jmeline! :100: :tada: 

Can you just fix the tests so I merge this? :sweat_smile: 

Totally amazing work here and thanks for your patience. :1st_place_medal:

**Comment by jmeline** [comment]
@anshumanv Of course 😸

**Comment by anshumanv** [comment]
I'm done with my traveling so I can get back real quick now! :rofl:

**Comment by jmeline** [comment]
All done @anshumanv

**Review by anshumanv — APPROVED** [review]
Thanks @jmeline :smile:

**Pull request merged**